### PR TITLE
[CONSOLE] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Improvements based on recent Romanian translation document revision https://github.com/reactos/reactos/commit/06e89b2e81cd7de029fac2c595a6371eba13d797.

This files was not translated in Romanian language in any Win version (neither XP/2k3+, nor in the previous ones). I tried to translate this file as close as possible in the way I did in the previous PRs with this attendum.

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian for review

File: dll/cpl/console/lang/ro-RO.rc [hunk 1]
```diff
@@ -14,27 +14,27 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_CAPTION
...
-    AUTORADIOBUTTON "&Mărită", IDC_RADIO_LARGE_CURSOR, 15, 60, 90, 10, WS_CHILD | WS_VISIBLE | WS_TABSTOP
-    GROUPBOX "Mod de afișare", IDC_STATIC, 133, 7, 112, 70, WS_CHILD | WS_VISIBLE | WS_GROUP
-    AUTORADIOBUTTON "În fere&astră", IDC_RADIO_DISPLAY_WINDOW, 142, 20, 70, 10, WS_CHILD | WS_VISIBLE | WS_TABSTOP
-    AUTORADIOBUTTON "Pe t&ot ecranul", IDC_RADIO_DISPLAY_FULL, 142, 40, 70, 10, WS_CHILD | WS_VISIBLE | WS_TABSTOP
-    GROUPBOX "Istoric pentru comenzi", IDC_STATIC, 7, 85, 120, 77, WS_CHILD | WS_VISIBLE | WS_GROUP
-    LTEXT "Ma&xim înregistrări:", IDC_STATIC, 14, 101, 70, 12
+    GROUPBOX "Dimensiunea cursorului", IDC_STATIC, 7, 7, 120, 70, WS_CHILD | WS_VISIBLE | WS_GROUP
+    AUTORADIOBUTTON "&Mică", IDC_RADIO_SMALL_CURSOR, 15, 20, 90, 10, WS_CHILD | WS_VISIBLE | WS_TABSTOP
+    AUTORADIOBUTTON "M&edie", IDC_RADIO_MEDIUM_CURSOR, 15, 40, 90, 10, WS_CHILD | WS_VISIBLE | WS_TABSTOP
+    AUTORADIOBUTTON "&Mare", IDC_RADIO_LARGE_CURSOR, 15, 60, 90, 10, WS_CHILD | WS_VISIBLE | WS_TABSTOP
```
Comment (JoachimHenze) [inline dll/cpl/console/lang/ro-RO.rc:20]: **mandatory** change requested: accel collision &Mare vs. &Mica

**Review by JoachimHenze — CHANGES_REQUESTED** [review]
see the collisions

File: dll/cpl/console/lang/ro-RO.rc (hunk 1 above)
Comment (AndreiMiloiu) [reply dll/cpl/console/lang/ro-RO.rc]: Fixed!

**Comment by AndreiMiloiu** [comment]
> see the collisions

Fixed now!

**Comment by AndreiMiloiu** [comment]
@GeoB99 is it good now?

**Review by HBelusca — APPROVED** [review]
Looks ok from a pure technical point of view.
@AndreiMiloiu Have you checked your modifications correctly fit into the controls?
Cc @GeoB99 for review.

**Comment by AndreiMiloiu** [comment]
> Looks ok from a pure technical point of view. @AndreiMiloiu Have you checked your modifications correctly fit into the controls? Cc @GeoB99 for review.

Yes

**Comment by AndreiMiloiu** [comment]
Any news?

**Pull request merged**